index.html

```diff
@@ -45,7 +45,185 @@ <h1 class="banner_content">I’m Munish. <br>Glad to see you!</h1>
+                        </ul>
+                        <a href="index.html" class="work_link">
+                            <button class="work_button">See project</button>
+                        </a>
+                    </div>
+                </div>
+            </div>  
+        </section>
+
+        <!-- Works Section Ends Here -->
+
+        <!-- About Section Starts Here -->
+
+        <section class="about">
+            <div class="about_content_wrapper">
+                <h3 class="about_content">About Myself</h3>
+                <p class="about_content">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <div class="about_social_box">
+                    <p class="lets_connect">LET'S CONNECT</p>
+                    <ul class="banner_social_icons_wrapper">
+                        <li><a href="index.html"><img class="banner_social_icons" src="images/twitter-icon.svg" alt="Twitter"></a></li>
+                        <li><a href="index.html"><img class="banner_social_icons" src="images/linkedin-icon.svg" alt="Linkedin"></a></li>
+                        <li><a href="index.html"><img class="banner_social_icons" src="images/social-icon-3.svg" alt="Social"></a></li>
+                        <li><a href="index.html"><img class="banner_social_icons" src="images/social-icon-4.svg" alt="Social"></a></li>
+                        <li><a href="index.html"><img class="banner_social_icons" src="images/social-icon-5.svg" alt="Social"></a></li>
+                    </ul>
+                </div>
+                <a href="index.html" class="resume_button"><button class="resume_button">Get my resume</button></a>
+            </div>
+            <div class="skills_wrapper">
+                <div class="dropdown_wrapper">
+                    <div class="dropdown_arrow_wrapper">
+                        <h4 class="dropdown_title">Languages</h4>
+                        <img src="images/down_arrow.png" alt="Down Arrow" class="dropdown_arrow">
+                    </div>
+                    <div class="dropdown_content_wrapper open">
+                        <div class="dropdown_content">
+                            <img class="skills" src="images/javascript-icon.png" alt="Skills">
+                            <p class="skills">JavaScript</p>
+                        </div>
+                        <div class="dropdown_content">
+                            <img class="skills" src="images/html-icon.png" alt="Skills">
+                            <p class="skills">HTML</p>
+                        </div>
+                        <div class="dropdown_content">
+                            <img class="skills" src="images/css-icon.png" alt="Skills">
+                            <p class="skills">CSS</p>
+                        </div>
```

Comment:
- [ ] Kindly, consider using `ul` to wrap the list of languages (e.g., "JavasScript", "HTML" and "CSS") and make sure each language wraps in a `li`. Remember you should always use `ul` and `li` to list elements. This will improve greatly the semantic of your webpage 😄 which will make your website much more accessible (for users, screen readers, and search engines).